src/undead/regexp.d

```diff
@@ -208,7 +208,7 @@ string sub(string s, string pattern, string format, string attributes = null)
 {
     auto r = new RegExp(pattern, attributes);
     auto result = r.replace(s, format);
-    delete r;
+    destroy(r);
```

Comment:
You can use `__delete` as a direct replacement for `delete`: https://dlang.org/phobos/core_memory#.__delete. I wonder if it's worth guarding this in `static if (__VERSION__ >= 2079)`.